showcase/gapic-showcase/src/test/java/com/google/showcase/v1beta1/ComplianceClientHttpJsonTest.java

```diff
@@ -50,20 +50,20 @@
 import java.util.HashMap;
 import java.util.List;
 import javax.annotation.Generated;
-import org.junit.After;
-import org.junit.AfterClass;
-import org.junit.Assert;
-import org.junit.Before;
-import org.junit.BeforeClass;
-import org.junit.Test;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
 
 @Generated("by gapic-generator-java")
-public class ComplianceClientHttpJsonTest {
+class ComplianceClientHttpJsonTest {
   private static MockHttpService mockService;
   private static ComplianceClient client;
 
-  @BeforeClass
-  public static void startStaticServer() throws IOException {
+  @BeforeAll
+  static void startStaticServer() throws IOException {
```

Comment:
IIRC, these unit test classes (i.e. ComplianceClientHttpJsonTest, ComplianceClientTest, etc..) are generated by the generator. I think these files will be overwritten on the next showcase regeneration so we'll probably need to coordinate a change in the generator + shared-deps/pom-parent for these.

Reply:
Yes, let's only migrate the integration tests in `it` folder.

Reply:
> If it doesn't exist already, can you create a ticket to update the generator to use JUnit5 when generating these unit tests?

I purposely excluded generated unit tests from the scope at this moment, because the change would apply to google-cloud-java and all handwritten libraries, we need to coordinate the change and update their poms accordingly. Also since the main benefit is to make unit testing in `sdk-platform-java` easier, I think it's better to exclude the effort at this moment.
On the other hand, creating a ticket to track the future effort is always a good idea.

Reply:
Good point. Did not realize these are generated tests. Will revert the changes. 

Created https://github.com/googleapis/sdk-platform-java/issues/2762 to track updating generator for unit tests gen.